azure-pipelines.yml

```diff
@@ -164,13 +207,19 @@ jobs:
   - script: if [%INSTALL_PICKLE5%]==[1] python -m pip install pickle5
     displayName: 'Install optional pickle5 backport (only for python3.6 and 3.7)'
   - powershell: |
+      #$env:PATH = $env:PATH + ";" +   "C:\\hostedtoolcache\\windows\\Python\\$pyversion\\$(PYTHON_ARCH)\\lib"
       If ($(BITS) -eq 32) {
          $env:NPY_DISTUTILS_APPEND_FLAGS = 1
          $env:CFLAGS = "-m32"
          $env:LDFLAGS = "-m32"
          $env:PATH = "C:\\tools\\mingw32\\bin;" + $env:PATH
          refreshenv
       }
+      # include path for openblas headers copied in above:
+      $pyversion = python -c "from __future__ import print_function; import sys; print(sys.version.split()[0])"
+      $target_include = Resolve-Path "C:\\hostedtoolcache\\windows\\Python\\$pyversion\\$(PYTHON_ARCH)\\include\\" | Select -ExpandProperty Path
+      $env:INCLUDE = $target_include + ";" + $env:INCLUDE 
+      echo $env:INCLUDE
```

Comment:
this doesn't seem to help with the symbol resolution, but was worth a try I guess